frame/election-provider-support/solution-type/src/single_page.rs

```diff
@@ -134,6 +134,12 @@ pub(crate) fn generate(def: crate::SolutionDef) -> Result<TokenStream2> {
 						}
 					}
 				};
+
+				// Make sure that the voter bound is binding.
+				use frame_support::traits::Get;
+				if #struct_name.voter_count() as u32 > #max_voters::get() {
```

Comment:
should 
1. re-export `Get` from `_fpes`
2. use fully qualified syntax.

In general, any import in macros should be strictly avoided, because you are assuming that some crate (`frame-support`) is in scope, so using the macro anywhere that this does not hold will break.